iron-form.html

```diff
@@ -417,29 +417,51 @@
       },
 
       _findElements: function(parent, ignoreName) {
-        var nodes = Polymer.dom(parent).querySelectorAll('*');
+        var node = parent.firstElementChild;
         var submittable = [];
 
-        for (var i = 0; i < nodes.length; i++) {
-          var node = nodes[i];
+        // This algorithm is O(3n) in terms of nodes. It traverses depth-first
```

Comment:
`O(3n)` makes no sense in terms of big O notation, see https://en.wikipedia.org/wiki/Big_O_notation#Multiplication_by_a_constant

It should be just `O(n)`